spec/oppiabotSpec.js

```diff
@@ -0,0 +1,85 @@
+require('dotenv').config();
+const {createRobot} = require('probot');
+const plugin = require('..');
```

Comment:
Write a comment above this to explain what plugin is, since it's not very descriptive here.

Reply:
Done, thanks!